ports/nrf/boards/pca10056/mpconfigboard_s140.mk

```diff
@@ -0,0 +1,8 @@
+MCU_SERIES = m4
+MCU_VARIANT = nrf52
+MCU_SUB_VARIANT = nrf52840
+SOFTDEV_VERSION = 6.0.0-6.alpha
```

Comment:
Adding a `?=` like this: `SOFTDEV_VERSION ?= 2.0.1` will make it easier to override this from the command line.

Reply:
Will do!